projects/unix/Makefile

```diff
@@ -439,7 +440,7 @@ SOURCE = \
 	$(SRCDIR)/pi/pi_controller.c \
 	$(SRCDIR)/pi/sram.c \
 	$(SRCDIR)/plugin/emulate_game_controller_via_input_plugin.c \
-	$(SRCDIR)/plugin/emulate_speaker_via_audio_plugin.c \
+	$(SRCDIR)/plugin/audio_backend_compat.c \
```

Comment:
Is it silly to have a distinct folder `plugin` and `backend`? As the new _backend_ approach is supposed to take away the concept of _plugin_, I just ask.